### ISSUE TITLE
Custom Prompt is not taking effect

### ISSUE THREAD

**Issue by junaidp** [ragapp/ragapp#32]
**Describe the bug**
Prompt is not taking effect

**To Reproduce**
Just add a new prompt .
Click update 
Send a New Query

**Expected behavior**
Response should take the effect of the Custom prompt, But there is no change
So i am confused ,if my prompts are going through, Because whatever i update in the prompt , there is no effect, Even a simple prompts , As in screenshot

**Screenshots**
<img width="1442" alt="image" src="https://github.com/ragapp/ragapp/assets/747843/921d7512-6afc-4df6-aa16-05f29af32cdb">

<img width="1619" alt="image" src="https://github.com/ragapp/ragapp/assets/747843/ceda1efa-7728-4ef2-bceb-1832e8fb3d6b">

**Comment by marcusschiesser** [comment]
@junaidp which LLM did you use in your test? it might not work as expected with system prompts

**Comment by junaidp** [comment]
I am using OPEN AI

**Comment by marcusschiesser** [comment]
@junaidp Thanks, already fixed on main branch. For testing the main branch, check: https://github.com/ragapp/ragapp/pkgs/container/ragapp

**Comment by leehuwuj** [comment]
Hi @junaidp ,I just tried out your custom prompt and it worked well.
<img width="1333" alt="image" src="https://github.com/ragapp/ragapp/assets/39040748/19355833-73b2-4f5b-884e-cc87e46093b3">

Please note that we haven't released the update to docker hub yet, but you're still able to test the change by using our image in github registry: https://github.com/ragapp/ragapp/pkgs/container/ragapp . Basically, you just need change the image name to: `ghcr.io/ragapp/ragapp:latest`
Eg:
```shell
docker run --rm -p 8000:8000 -it ghcr.io/ragapp/ragapp:latest
```

**Comment by junaidp** [comment]
ah yes , works now , Thanks !

**Comment by marcusschiesser** [comment]
reopening as not released yet

**Comment by marcusschiesser** [comment]
fixed in v0.0.9
